protocols/protocols.go

```diff
@@ -38,7 +38,7 @@ import (
 	_ "github.com/dedis/cothority/protocols/randhound"
 	// ByzCoin has some strange library which uses 'seelog' that doesn't
 	// free all go-routines
-	//_ "github.com/dedis/cothority/protocols/byzcoin"
-	//_ "github.com/dedis/cothority/protocols/byzcoin/ntree"
-	//_ "github.com/dedis/cothority/protocols/byzcoin/pbft"
+	_ "github.com/dedis/cothority/protocols/byzcoin"
```

Comment:
There was a reason we commented these 3 lines out (see comment on top). I think we should keep it this way and add a comment in the ByzCoin simulation pointing to these lines?